.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "submodules/libadm"]
 	path = submodules/libadm
 	url = https://github.com/ebu/libadm.git
-	branch = eps
+	branch = eps_2076-2
```

Comment:
Just making sure this doesn't need https://github.com/ebu/libadm/commits/time_add_subtract_operators? I can't see anywhere you would need it anymore as you convert to nanoseconds in your approach.